__tests__/app.test.js

```diff
@@ -234,4 +234,13 @@ describe("GET /api/articles/:article_id/comments", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+
+  test("responds with 200 status even if there are no comments for the specified article (empty array)", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(0);
```

Comment:
Nearly there. This test only checks for a length of 0, which an empty string would also have. Can we add in a check that is actually an array please.